geometry_msgs/PoseStamped initial_cam_pose
geometry_msgs/PoseStamped target_pose
geometry_msgs/Vector3Stamped up
---
geometry_msgs/PoseStamped new_cam_pose
